lang/en/observation.php

```diff
@@ -40,6 +40,7 @@
 $string['defaultmessagenoinstructions'] = 'No instructions provided.';
 $string['timeslots'] = 'Timeslots';
 $string['observationpoints'] = 'Observation Points';
+$string['observationpoint'] = 'Observation Point';
```

Comment:
General comment, moodle is usually sentence case, eg 'Observation point' vs the above 'Observation Point'